app/models/agents/java_script_agent.rb

```diff
@@ -114,12 +114,9 @@ def execute_js(js_function, incoming_events = [])
       context["getOptions"] = lambda { |a, x| interpolated.to_json }
       context["doLog"] = lambda { |a, x| log x }
       context["doError"] = lambda { |a, x| error x }
-      context["getMemory"] = lambda do |a, x, y|
```

Comment:
I don't know if how much more efficient it would be, but would it make sense to keep `getMemory` with one argument to not always pass the whole memory through the javascript bridge?